controls/pcidss_ocp4.yml

```diff
@@ -1624,41 +1647,58 @@ controls:
   description: 'This access control system must include the following:'
   levels:
   - base
-  notes: ''
-  status: pending
+  notes: |-
+      The OpenShift Container Platform has RBAC enabled by default. With this
+      feature, any requests to the API is denied unless an explicit
+      Role or ClusterRule is bound to the entity making the request.
+  status: inherently met
   controls:
   - id: Req-7.2.1
     title: 7.2.1 Coverage of all system components
     levels:
     - base
-    notes: ''
-    status: pending
+    notes: |-
+      The OpenShift Container Platform has RBAC enabled by default. With this
+      feature, any requests to the API is denied unless an explicit
+      Role or ClusterRule is bound to the entity making the request.
+
+      All objects in Kubernetes/OpenShift are bound to RBAC permissions.
+    status: inherently met
```

Comment:
More of a question about "inherently met" - this really means that "by virtue of the fact (note details) this is taken care of, but this is not automated